bigbluebutton-html5/imports/ui/components/app/styles.js

```diff
@@ -49,54 +52,41 @@ const DtfInvert = `
   .tl-container {
     background-color: var(--tl-background) !important;
   }
```

Comment:
This is needed to fix background color in pdfs with transparent background 
```suggestion
  .tl-container {
    .tl-image {
      background-color: white !important;
    }
  }
```